supabase/functions/backend/controllers/BroadcastController.ts

```diff
@@ -1,60 +1,69 @@
-	await BroadcastService.make()
-	return AppResponse.ok(res, {}, 204)
+async function makeBroadcast(_req: Request, res: Response) {
+  await BroadcastService.make()
+  return AppResponse.ok(res, {}, 204)
 }
 
 async function sendDraft(req: Request, res: Response) {
-	const validations = [
-		param('broadcastID').isInt().toInt(),
-		query('isSecond').optional().isBoolean().toBoolean(),
-	]
-	await validateAndResponse(validations, req, res)
-
-	const id = Number(req.params.broadcastID)
-	const { isSecond } = req.query
+  const validations = [
+    param('broadcastID').isInt().toInt(),
+    query('isSecond').optional().isBoolean().toBoolean(),
+  ]
+  await validateAndResponse(validations, req)
 
-	await BroadcastService.sendBroadcastMessage(id, Boolean(isSecond))
-	return AppResponse.ok(res)
+  const id = Number(req.params.broadcastID)
+  const { isSecond } = req.query
+  await BroadcastService.sendBroadcastMessage(id, Boolean(isSecond))
+  return AppResponse.ok(res)
 }
 
 async function getAll(req: Request, res: Response) {
-	const validations = [
-		query('limit').optional().isInt().toInt(),
-		query('cursor').optional().isInt().toInt(),
-	]
-	await validateAndResponse(validations, req, res)
-	const { limit, cursor } = req.query
-	const result = await BroadcastService.getAll(limit, cursor)
-	return AppResponse.ok(res, result)
+  const validations = [
+    query('limit').optional().isInt().toInt(),
+    query('cursor').optional().isInt().toInt(),
+  ]
+  await validateAndResponse(validations, req)
+  const { limit, cursor } = req.query
+  const result = await BroadcastService.getAll(limit, cursor)
+  return AppResponse.ok(res, result)
 }
 
 async function patch(req: Request, res: Response) {
-	const validations = [
-		param('id').isInt().toInt(),
-		body('firstMessage').optional().isString().notEmpty(),
-		body('secondMessage').optional().isString().notEmpty(),
-		body('runAt').optional().isDecimal(),
-		body('delay').optional().isString().notEmpty(),
-	]
-
-	await validateAndResponse(validations, req, res)
-	const id = Number(req.params.id)
-	const broadcast: BroadcastUpdate = req.body
-
-	const result = await BroadcastService.patch(id, broadcast)
-	return AppResponse.ok(res, result)
+  const validations = [
+    param('id').isInt().toInt(),
+    body('firstMessage').optional().isString().notEmpty(),
+    body('secondMessage').optional().isString().notEmpty(),
+    body('runAt').optional().isDecimal(),
+    body('delay').optional().isString().notEmpty(),
+  ]
+
+  await validateAndResponse(validations, req)
+  const id = Number(req.params.id)
+  const broadcast: BroadcastUpdate = req.body
+  const result = await BroadcastService.patch(id, broadcast)
+  return AppResponse.ok(res, result)
+}
+
+async function updateTwilioStatus(req: Request, res: Response) {
+  const validations = [
+    param('broadcastID').isInt().toInt(),
+  ]
+  await validateAndResponse(validations, req)
+  const id = Number(req.params.broadcastID)
```

Comment:
If `toInt` is already presented, why do you still need to call `Number(req.params.broadcastID)` 🤔?